src/main/java/mServer/crawler/sender/phoenix/tasks/PhoenixFilmDetailTask.java

```diff
@@ -0,0 +1,151 @@
+package mServer.crawler.sender.phoenix.tasks;
+
+import com.google.gson.reflect.TypeToken;
+import de.mediathekview.mlib.daten.DatenFilm;
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.net.MalformedURLException;
+import java.time.format.DateTimeFormatter;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import javax.ws.rs.client.WebTarget;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.parsers.SAXParser;
+import javax.xml.parsers.SAXParserFactory;
+import mServer.crawler.CrawlerTool;
+import mServer.crawler.sender.MediathekReader;
+import mServer.crawler.sender.MediathekZdf;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'mServer.crawler.sender.MediathekZdf'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)